tests/test_codex_wrapper_computer_use_mcp.py

```
@@ -879,54 +1010,282 @@ def test_codex_gets_cmux_cua(failures: list[str]) -> None:
+        f"unrelated symlink must remain untouched, got {skill}",
+        failures,
+    )
+    expect(
+        configured_skill_path(args) is None,
+        f"preserved user link must not receive a duplicate fallback, got {args}",
+        failures,
+    )
+
+
+def test_codex_preserves_unverified_legacy_computer_use_link(failures: list[str]) -> None:
     code, args, stderr, skill = run_wrapper(["hello"], preexisting_legacy_link=True)
```

Ruff (0.16.3)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)